dropwizard-jetty/src/test/resources/logback-test.xml

```diff
@@ -1,13 +1,3 @@
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-        </encoder>
-    </appender>
-
-    <logger level="info" name="io.dropwizard.jetty.HttpsConnectorFactory"/>
```

Comment:
I think this would cause changed behaviour. The appender would be inherited from the root logger, but this logger would actually log, wouldn't it?

Reply:
It was added in aabb36660401e32b2a7aa60a3320f12ad59ece1b which added a test for building the HTTPS connector.

The only thing really of interest which gets logged when doing that are the SSL parameters, but we don't need those logs for the test - it checks what the HTTPS connector actually gets configured with.

Reply:
Agreed